src/assets/translations/en.json

```diff
@@ -127,6 +127,7 @@
     "listingNotUpdated": "Your listing was not updated. Please try again!",
     "listings": "Listings",
     "listingUpdatedAdmin": "Your listing is uploaded and active. Redirecting to dashboard in 5 seconds.",
+    "listingCreated": "Your listing was created succesfully. It is under review by Admin and will be updated soon. Redirecting to dashboard in 5 seconds.",
```

Comment:
remove It is under review by Admin and will be updated soon. in **listingUpdated** section